pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -381,5 +382,5 @@ func (f *Fit) Score(ctx context.Context, state *framework.CycleState, pod *v1.Po
 		}
 	}
 
-	return f.score(pod, nodeInfo, s.podRequests)
+	return f.score(klog.FromContext(ctx), pod, nodeInfo, s.podRequests)
```

Comment:
```suggestion
	return f.score(ctx, pod, nodeInfo, s.podRequests)
```
Prefer context than logger as parameters.

Reply:
Pass context as a parameter is also ok. Updated.